bin/test-workflow/8_app_deploy.sh

```diff
@@ -10,22 +10,38 @@ source utils.sh
 check_env_var TEST_APP_NAMESPACE_NAME
 check_env_var CONJUR_AUTHN_LOGIN_PREFIX
 
-announce "Deploying summon-sidecar test app in $TEST_APP_NAMESPACE_NAME."
-
 set_namespace "$TEST_APP_NAMESPACE_NAME"
 
-# Uninstall sample app if it exists
-if [ "$(helm list -q -n $TEST_APP_NAMESPACE_NAME | grep "^app-summon-sidecar$")" = "app-summon-sidecar" ]; then
+pushd ../../helm/conjur-app-deploy > /dev/null
+
+  announce "Deploying summon-sidecar test app in $TEST_APP_NAMESPACE_NAME."
+  # Uninstall sample app if it exists
+  if [ "$(helm list -q -n $TEST_APP_NAMESPACE_NAME | grep "^app-summon-sidecar$")" = "app-summon-sidecar" ]; then
     helm uninstall app-summon-sidecar -n "$TEST_APP_NAMESPACE_NAME"
-fi
+  fi
 
-pushd ../../helm/conjur-app-deploy > /dev/null
   helm install app-summon-sidecar . -n "$TEST_APP_NAMESPACE_NAME" --debug --wait --timeout "$TIMEOUT" \
+      --render-subchart-notes \
       --set global.conjur.conjurConnConfigMap="conjur-connect" \
       --set app-summon-sidecar.enabled=true \
       --set app-summon-sidecar.conjur.authnLogin="$CONJUR_AUTHN_LOGIN_PREFIX/test-app-summon-sidecar" \
       --set app-summon-sidecar.app.image.tag="$CONJUR_NAMESPACE_NAME" \
-      --set app-summon-sidecar.app.image.repository="$DOCKER_REGISTRY_PATH/test-sidecar-app"
+      --set app-summon-sidecar.app.image.repository="$DOCKER_REGISTRY_PATH/test-sidecar-app" \
+      --set app-summon-sidecar.conjur.authnConfigMap.name="conjur-authn-configmap-summon-sidecar"
+
+
+  announce "Deploying secretless-broker test app in $TEST_APP_NAMESPACE_NAME."
+  # Uninstall sample app if it exists
+  if [ "$(helm list -q -n $TEST_APP_NAMESPACE_NAME | grep "^app-secretless-broker$")" = "app-secretless-sidecar" ]; then
+    helm uninstall app-secretless-broker -n "$TEST_APP_NAMESPACE_NAME"
+  fi
+
+  helm install app-secretless-broker . -n "$TEST_APP_NAMESPACE_NAME" --debug --wait --timeout "$TIMEOUT" \
```

Comment:
Is it possible to install both sample apps with one `helm install` command with both `--set app-summon-sidecar.enabled=true` and `--set app-secretless-broker.enabled=true` values set?